app/views/admin/events/index.html.haml

```diff
@@ -14,24 +16,42 @@
                 %li= link_to 'All Events', admin_conference_program_events_path(@conference.short_title, format: :pdf, event_export_option: 'all')
                 %li= link_to 'Confirmed Events', admin_conference_program_events_path(@conference.short_title, format: :pdf, event_export_option: 'confirmed')
                 %li= link_to 'All Events with Comments', admin_conference_program_events_path(@conference.short_title, format: :pdf, event_export_option: 'all_with_comments')
-              .btn-group
-                %button.btn.btn-default.dropdown-toggle{"data-toggle" => "dropdown", :type => "button", :class => 'btn btn-success'}
-                  Export CSV
-                  %span.caret
-                %ul.dropdown-menu{:role => "menu"}
-                  %li= link_to 'All', admin_conference_program_events_path(@conference.short_title, format: :csv, event_export_option: 'all')
-                  %li= link_to 'Confirmed', admin_conference_program_events_path(@conference.short_title, format: :csv, event_export_option: 'confirmed')
-                  %li= link_to 'All with Comments', admin_conference_program_events_path(@conference.short_title, format: :csv, event_export_option: 'all_with_comments')
-              .btn-group
-                %button.btn.btn-default.dropdown-toggle{"data-toggle" => "dropdown", :type => "button",  :class => 'btn btn-success'}
-                  Export XLS
-                  %span.caret
-                %ul.dropdown-menu{:role => "menu"}
-                  %li= link_to 'All', admin_conference_program_events_path(@conference.short_title, format: :xlsx, event_export_option: 'all')
-                  %li= link_to 'Confirmed', admin_conference_program_events_path(@conference.short_title, format: :xlsx, event_export_option: 'confirmed')
-                  %li= link_to 'All with Comments', admin_conference_program_events_path(@conference.short_title, format: :xlsx, event_export_option: 'all_with_comments')
+            .btn-group
```

Comment:
Because 'PDF Export' button was not in btn-group with the others.
Before:
![screenshot from 2017-01-10 13-12-23](https://cloud.githubusercontent.com/assets/4322737/21804773/5ce4d02c-d739-11e6-8470-c85a48be15d3.png)
After:
![screenshot from 2017-01-10 13-10-35](https://cloud.githubusercontent.com/assets/4322737/21804779/64a50ca0-d739-11e6-9db2-2d1469cf0f22.png)